langchain/package.json

```diff
@@ -843,7 +846,7 @@
     "@aws-sdk/credential-provider-node": "^3.388.0",
     "@aws-sdk/types": "^3.357.0",
     "@azure/storage-blob": "^12.15.0",
-    "@clickhouse/client": "^0.0.14",
+    "@clickhouse/client": "^0.2.5",
```

Comment:
Will this affect the other package depending on this?

Reply:
We've tested and confirmed that the package is exclusively used in the MyScale vector store (before Clickhouse support) and has passed the MyScale integration test. We believe it will not impact other packages.